ckanext/external_storage/plugin.py

```diff
@@ -23,15 +23,16 @@ class ExternalStoragePlugin(plugins.SingletonPlugin):
     def update_config(self, config):
         toolkit.add_template_directory(config, 'templates')
         toolkit.add_public_directory(config, 'public')
-        toolkit.add_resource('fanstatic', 'external-storage')
+        toolkit.add_resource('webassets', 'external_storage')
 
     # ITemplateHelpers
 
     def get_helpers(self):
         return {'extstorage_server_url': helpers.server_url,
-                # 'extstorage_resource_authz_scope': helpers.resource_authz_scope,
-                # 'extstorage_resource_storage_prefix': helpers.resource_storage_prefix,
-                'extstorage_storage_namespace': helpers.storage_namespace}
+                'extstorage_resource_authz_scope': helpers.resource_authz_scope,
+                'extstorage_resource_storage_prefix': helpers.resource_storage_prefix,
+                'extstorage_storage_namespace': helpers.storage_namespace,
+                'extstorage_use_legacy_file_uploader_ui': True}
```

Comment:
Is this a good place to set the `extstorage_use_legacy_file_uploader_ui` variable ?
Or should we be storing the config elsewhere

Reply:
No this isn't quite right, we want the config parm to be set in the adx_config.ini file. the get_helpers function is to return a list of Jinja2 template helpers (python functions that can be called from the template syntax).  You're not far off though, you definitely should use the helper to pass the config param to the template. You're just hard coding it as True in the python code at the moment, instead of fetching it from the config file. 

A great example to follow is the extstorage_storage_namespace helper.  That helper, defined in `helpers.storage_namespace()` is set up for the sole purpose of passing a config param from the `adx_config.ini` file to the `templates/external_storage/upload_module.html` template.